clang-tools-extra/clang-doc/MDGenerator.cpp

```diff
@@ -57,15 +59,22 @@ static void writeFileDefinition(const ClangDocContext &CDCtx, const Location &L,
     OS << "*Defined at " << L.Filename << "#" << std::to_string(L.LineNumber)
        << "*";
   } else {
-    OS << "*Defined at [" << L.Filename << "#" << std::to_string(L.LineNumber)
-       << "](" << StringRef{*CDCtx.RepositoryUrl}
-       << llvm::sys::path::relative_path(L.Filename) << "#"
-       << std::to_string(L.LineNumber) << ")"
-       << "*";
+
+    OS << formatv("*Defined at [#{0}{1}{2}](#{0}{1}{3})*",
+                  CDCtx.RepositoryLinePrefix.value_or(""),
+                  std::to_string(L.LineNumber), L.Filename,
```

Comment:
It's a format string, you don't need to convert the number to a string, the formatter will handle that, right? There are additional format specifiers if you need them (which you shouldn't). This is also how you handled it above.